server/svix-server/src/worker.rs

```diff
@@ -80,6 +84,8 @@ async fn dispatch(
     payload: &Json,
     msg_task: MessageTask,
     endp: CreateMessageEndpoint,
+    create_message_app: CreateMessageApp,
+    op_webhook_sender: Option<&OperationalWebhookSender>,
```

Comment:
I wouldn't make it optional, but rather just make it a no-op to send.
Yeah, I know you'll have a few extra clones when passing values to it, but honestly, it's not a biggie. We can always just change the structures to use references instead of strings if this ever becomes a concern.

To be honest though, we probably want the struct to use lifetimes and references. It'll save you from all of the clones and code changes you had to do.